nixos/modules/services/x11/urxvtd.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+# maintainer: siddharthist
+
+with lib;
+
+let
+  cfg = config.services.urxvtd;
+in {
+
+  options.services.urxvtd.enable = mkOption {
+    type = types.bool;
+    default = false;
+    example = true;
+    description = ''
+      Enable urxvtd, the urxvt terminal daemon. To use urxvtd, run
+      export RXVT_SOCKET=/run/user/your-uid/urxvtd-socket and then
```

Comment:
Right, don't set `default.target` or it will start automatically for all the users (even if not logged).
Also the is no `graphical.target` for user services:
```
λ systemctl --user list-units --type target                                                                                                                          130 ↵UNIT           LOAD   ACTIVE SUB    DESCRIPTION
basic.target   loaded active active Basic System
default.target loaded active active Default
paths.target   loaded active active Paths
sockets.target loaded active active Sockets
timers.target  loaded active active Timers
```